### ISSUE TITLE
Need a quick clarification regarding decision to use RNA assay for DE after SCTransform

### ISSUE THREAD

**Issue by CodeInTheSkies** [sansomlab/tenx#55]
Hello there,

After a lot of confusion about whether to use the RNA assay or the SCT assay for gene-level analyses (like DE for example), I came across your repository (https://github.com/sansomlab/tenx/commit/d2e866d1c6e6ca23dcce9a481e6cbf79a1092773), which did a great job summarizing the current advise from the Seurat authors on this topic!  Thanks a lot for making this public.

I would like to ask you, what is your opinion on using `FindAllMarkers` after SCT like the **_"option 1"_** here:  https://github.com/satijalab/seurat/issues/1501

In other words, one of the recommendations of @satijalab seems to be that we can use the SCT assay's `@data` slot for `FindAllMarkers`.  Of course, they do say that the "absolute best" way is to directly use the Pearson residuals from `@scale.data` slot, but then the avg_logFC doesn't have the desired meaning anymore.  So, unless we have updated vignettes, they don't fully support it.

But I would like to ask your opinion on using the `@data` slot.

Thanks a lot!

**Comment by CodeInTheSkies** [comment]
I would like to add another point to my question. The description of SCTransform says:

> This function calls sctransform::vst. The sctransform package is available at https://github.com/ChristophH/sctransform. Use this function as an alternative to the NormalizeData, FindVariableFeatures, ScaleData workflow. Results are saved in a  new assay (named SCT by default) with counts being (corrected) counts, data being log1p(counts), scale.data being pearson residuals; sctransform::vst intermediate results are saved in misc slot of new assay. 

From the above and from here https://github.com/ChristophH/sctransform/issues/36#issuecomment-526056501, it is clear that the `@data` slot of the SCT assay contains the log-transformed corrected counts, and so the avg_logFC will make sense in this case.

**Comment by snsansom** [comment]
Dear CodeInTheSkies, 

Thanks for your interest and comments. I recommend to follow the Satija labs' advice (see e.g. https://github.com/satijalab/seurat/issues/1421). We still use the RNA counts for now.

Alternatively, if we have sufficient replication we generally adopt a pseudobulk approach. Recently this has been made easier by the availability of the muscat package (https://bioconductor.org/packages/devel/bioc/html/muscat.html), and see the bioRxiv paper here: https://www.biorxiv.org/content/10.1101/713412v1.

**Comment by LooLipin** [comment]
Hi @snsansom,

Do you Normalize and scale the data in "RNA" after integration before DE analysis?

I have been doing DE analysis on "SCT" data and it looks very different to the genes I get from "RNA". Please advise.
